puppylinux-woof-CE/woof-CE
# more options for puppyapps

-dc: replace defaults chooser [if found]
-fddb: download and store latest freedesktop.org.xml
-mimedb: update mime database using the puppyapps xml files
-mimedefs: create mime associations for different standards
-system: -dc + -mimedb + -mimedefs

to test and debug -mimedefs:
    export TEST=1
    puppyapps -mimedefs
    (now go to /tmp/mimedefs and check)
you can do this afterwards:
    puppyapps -mimedefsa &> mimedefs.log

----

These functions use files from /var/lib/puppyapps

Sample files:

mimedb/*.xml - mime packages
dotdesktop - custom .desktop files
exec - app:exec [how to run properly certain apps]
mime_associations - mime associations [mimetype=app;app2;app3]
mime_defaults - mime default associations

Entries/data can be added/altered/removed

Puppyapps can be used to experiment, test new versions of shared-mime-info and can be very useful
to the primitive puppy "desktop environment" making it a bit configurable and expandable
without the need to create more files, all are stored in "dabatases" in /var/lib/puppyapps

I use it with my own "databases" to generate all "my environment" when I do remasters.
